### ISSUE TITLE
[1.10.2] I don't know what happend :( Crash

### ISSUE THREAD

**Issue by Blatts12** [desht/ModularRouters#21]
I was afk and this happend.

https://pastebin.com/f59Wbxi5

**Comment by desht** [comment]
I'm away from my computer for the next few days, but I'll take a look when I can.

**Comment by desht** [comment]
This trace makes zero sense:
```
java.lang.NullPointerException: Exception while adding particle
    at net.minecraft.world.World.func_189509_E(World.java:218)
    at net.minecraft.world.World.func_175625_s(World.java:2387)
    at me.desht.modularrouters.block.tile.TileEntityItemRouter.getRouterAt(TileEntityItemRouter.java:855)
    at me.desht.modularrouters.proxy.ClientProxy.lambda$registerBlockColors$0(ClientProxy.java:132)
    at net.minecraft.client.renderer.color.BlockColors.func_186724_a(BlockColors.java:167)
    at net.minecraft.client.particle.ParticleDigging.func_187154_b(SourceFile:52)
```

There's no way that my ClientProxy's ``ClientProxy.lambda$registerBlockColors`` could be called like this.  That code is *only* called in the module's init phase, and most certainly never from any particle code.

Honestly no idea what happened here, but I suspect some other problem, maybe some kind of JVM corruption?  If it happens again let me know, but I see no way my code could have caused this.

**Comment by desht** [comment]
This is fixed in latest release (for all of 1.10.2, 1.11.2, 1.12/1.12.1) - see https://github.com/desht/ModularRouters/issues/24
